pkg/operation/botanist/component/metricsserver/metrics_server.go

```diff
@@ -92,8 +93,8 @@ type metricsServer struct {
 	image             string
 	vpaEnabled        bool
```

Comment:
With the new approach the `vpaEnabled` seems to be no longer used. You can clean it up.